pkgs/tools/inputmethods/ibus-engines/ibus-table-others/default.nix

```diff
@@ -2,22 +2,18 @@
 
 stdenv.mkDerivation rec {
   pname = "ibus-table-others";
-  version = "1.3.12";
+  version = "1.3.13";
 
   src = fetchurl {
     url = "https://github.com/moebiuscurve/ibus-table-others/releases/download/${version}/${pname}-${version}.tar.gz";
```

Comment:
Whats the difference to using the source direct?

Reply:
In principle, the currently used URL is the official source tarball provided by upstream and should be preferred.  In practice, for this release the author ran automake before creating the official tarball.

    Only in 1.3.13/tables: .gitignore
    Only in 1.3.13/tables: emoji-table.txt.data
    Only in 1.3.13: .gitignore
    Only in ibus-table-others-1.3.13/emoji-src: Makefile.in
    Only in ibus-table-others-1.3.13/icons: Makefile.in
    Only in ibus-table-others-1.3.13/m4: Makefile.in
    Only in ibus-table-others-1.3.13/tables: Makefile.in
    Only in ibus-table-others-1.3.13: Makefile.in
    Only in ibus-table-others-1.3.13: aclocal.m4
    Only in ibus-table-others-1.3.13: configure

Reply:
and ``emoji-table.txt.data``. If we would apply any patches we would need to run autoconf. I would prefer using the github source code.

Reply:
`emoji-table.txt.data` doesn't seem to belong in `tables/` for any reason, but `emoji-src/`, plus we don't patch this package and it has been stable for over a decade.  I am not convinced that the official source tarball upstream provide should not be preferred.